platform/viewer/src/components/Header/Header.js

```diff
@@ -61,10 +63,11 @@ function Header(props) {
     setOptions(optionsValue);
   }, [setOptions, show, t, user, userManager]);
 
-  const { appConfig = {} } = AppContext;
+  const { appConfig = {} } = useContext(AppContext);
   const showStudyList =
     appConfig.showStudyList !== undefined ? appConfig.showStudyList : true;
 
+  const listPath = RoutesUtil.parseStudyListPath(appConfig, server);
```

Comment:
This is an interesting point.
We may just want to "bake" in the study and list routes, where the list route is the "home" or "base" path.
(For our viewer implementation)

I'm trying to think of the implications here, but the only odd one is GCloud (for our usage).
We can talk through what this may need to look like with the v2 redesign, but I think you can nix the complexity here.

Reply:
ok I will do as we discussed: adding a homePath at appConfig level and default